### PR TITLE
Feature/con 209

### PR DESCRIPTION
**Pull request by alexcamposruiz**

### CODE REVIEW

**Comment by samuelraeburn** [comment]
Merge into develop not master

**Review by samuelraeburn — CHANGES_REQUESTED** [review]
update target branch

**Review by samuelraeburn — APPROVED** [review]

**Review by samuelraeburn — COMMENTED** [review]
Should we also update docs?
https://community.rti.com/static/documentation/connector/1.0.0/api/python/connector.html
I have just noticed that the link there to the XML Application Based Creation section of manual is broken.

File: rticonnextdds_connector/rticonnextdds_connector.py [hunk 1]
```diff
@@ -1224,7 +1224,7 @@ class Connector:
     A ``Connector`` instance must be deleted with :meth:`close()`.
 
     :param str config_name: The configuration to load. The ``config_name`` format is ``"LibraryName::ParticipantName"``, where ``LibraryName`` is the ``name`` attribute of a ``<domain_participant_library>`` tag, and ``ParticipantName`` is the ``name`` attribute of a ``<domain_participant>`` tag inside the library.
-    :param str url: An URL locating the XML document. The ``url`` can be a file path (for example, ``'/tmp/my_dds_config.xml'``) or a string containing the full XML document with the following format ``'str://"<dds>...</dds>"'``)
+    :param str url: An URL locating the XML document. The ``url`` can be a file path (for example, ``'/tmp/my_dds_config.xml'``), a string containing the full XML document with the following format ``'str://"<dds>...</dds>"'``), or a combination of multiple files or strings, as explained in the `URL Groups <https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_professional/users_manual/index.htm#users_manual/URL_Groups.htm>`__ section of the *Connext DDS Core Libraries User's Manual*.
```
Comment (samuelraeburn) [inline rticonnextdds_connector/rticonnextdds_connector.py:1227]: The link "https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_professional/users_manual/index.htm#users_manual/URL_Groups.htm" is broken

**Review by samuelraeburn — CHANGES_REQUESTED** [review]

**Comment by samuelraeburn** [comment]
Maybe this is related to CON-211?

File: rticonnextdds_connector/rticonnextdds_connector.py (hunk 1 above)
Comment (alexcamposruiz) [reply rticonnextdds_connector/rticonnextdds_connector.py:1227]: It's expected. This is the future doc dir structure in Hercules.

**Review by samuelraeburn — APPROVED** [review]

**Pull request merged**